app/src/main/res/layout/activity_in_app_share.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <android.support.constraint.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/white">
+
+        <include
+            android:id="@+id/toolbar_share_app"
+            layout="@layout/app_toolbar"
+            app:layout_constraintBottom_toTopOf="@id/scroll_view"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <!--<ProgressBar
+            android:id="@+id/appShare_ProgressBar"
+            android:layout_width="@dimen/width_70"
+            android:layout_height="@dimen/height_100"
+            android:indeterminate="true"
+            android:progressDrawable="@drawable/bg_progress_bar"
+            android:indeterminateTint="@color/colorPrimary"
+            android:indeterminateTintMode="src_atop"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />-->
+
+        <ScrollView
+            android:id="@+id/scroll_view"
+            style="@style/ScrollStyle"
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/toolbar_share_app">
+
+            <android.support.constraint.ConstraintLayout
+                android:id="@+id/layout_constraint"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_marginStart="@dimen/margin_30"
+                android:layout_marginLeft="@dimen/margin_30"
+                android:layout_marginTop="@dimen/margin_25"
+                android:layout_marginEnd="@dimen/margin_30"
+                android:layout_marginRight="@dimen/margin_30"
+                android:visibility="visible">
+
+
+                <TextView
+                    android:id="@+id/text_view_second_text"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/margin_30"
+                    android:fontFamily="@font/poppins_regular"
+                    android:gravity="center"
+                    android:text="@string/qr_code_text"
```

Comment:
we can add **tools:text** rather using android:text. So we don't need to bother about static string during runtime exception
ref : https://stackoverflow.com/questions/43002316/what-is-difference-between-toolstext-and-androidtext-in-android

Reply:
Thanks for sharing but this xml file is not exist in current codebase.